From d1c5e000131fa1cedcf5f17766758e498d043272 Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:25:04 -0400
Subject: [redhat] kabi: Adding symbol scsi_reset_provider (drivers/scsi/scsi_error.c)

Message-id: <3442074d33a031827059b3ff63960e4c87cb8581.1410510773.git.poros@redhat.com>
Patchwork-id: 96006
O-Subject: [PATCH RHEL7.1 184/233] kabi: Adding symbol scsi_reset_provider (drivers/scsi/scsi_error.c)
Bugzilla: 1071098
RH-Acked-by: Rob Evers <revers@redhat.com>

Bugzilla: 1071098
    https://bugzilla.redhat.com/show_bug.cgi?id=1071098

    Description:
    ============
    Diskres provides a SCSI bus reset, invoked by the DiskReservation
    agent for devices that don't respond well to the SCSI2 device RESET command.

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/scsi_reset_provider
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/scsi_reset_provider
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/scsi_reset_provider

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/scsi_reset_provider b/redhat/kabi/kabi-module/kabi_ppc64/scsi_reset_provider
new file mode 100644
index 0000000..109fea9
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/scsi_reset_provider
@@ -0,0 +1,2 @@
+#1-
+0x078e60ca scsi_reset_provider vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/scsi_reset_provider b/redhat/kabi/kabi-module/kabi_s390x/scsi_reset_provider
new file mode 100644
index 0000000..c10d19c
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/scsi_reset_provider
@@ -0,0 +1,2 @@
+#1-
+0xe4a4dae3 scsi_reset_provider vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/scsi_reset_provider b/redhat/kabi/kabi-module/kabi_x86_64/scsi_reset_provider
new file mode 100644
index 0000000..9369d3d
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/scsi_reset_provider
@@ -0,0 +1,2 @@
+#1-
+0x057104af scsi_reset_provider vmlinux EXPORT_SYMBOL
-- 
1.7.1